packages/sdk/src/getTransferData/getTransferData.ts

```diff
@@ -21,6 +21,7 @@ export interface GetTransferDataParams extends Partial<Signers> {
   transferConfig: TransferConfig;
 }
 
+// TODO: make getTransferData work with EvmSigner and PolkadotSigner
```

Comment:
is this not taken care of?

Reply:
remove the comment for now

Reply:
but what has to work with both Evm and Polkadot signer?

Reply:
it was a typo, I wanted to write without.